internal/command/check.go

```diff
@@ -30,17 +31,34 @@ func (r Runner) Check() {
 		log.Fatalf("failed to change dir: %v", err)
 	}
 
+	zombieChecker := zombie.NewChecker()
+	if err = zombieChecker.FindMocks(ctx); err != nil {
+		fmt.Fprintf(os.Stderr, "failed to find mock list generated by gomock: %v\n", err)
+	}
+
 	isFail := false
-	g, _ := errgroup.WithContext(context.Background())
+	g, _ := errgroup.WithContext(ctx)
 	for _, m := range ch.Mocks {
 		m := m
 
 		var runner mockgen.Runner
 		g.Go(func() error {
 			switch m.Mode {
 			case model.ReflectMode:
+				key := m.ReflectModeRunner.PackageName + " (interfaces: " + m.ReflectModeRunner.Interfaces + ")"
+				if ok := zombieChecker.Search(key); !ok {
+					fmt.Fprintf(os.Stderr, "[ERROR] mock is not up to date. destination: %s\n", key)
```

Comment:
It is good idea, we can find configured but un-generated mock. 👍 

nits:

```suggestion
					fmt.Fprintf(os.Stderr, "[ERROR] mock has not been generated. destination: %s\n", key)
```